src/topology/algebra/infinite_sum.lean

```diff
@@ -231,10 +236,25 @@ lemma summable_sum {f : γ → β → α} {s : finset γ} (hf : ∀i∈s, summab
   summable (λb, ∑ i in s, f i b) :=
 (has_sum_sum $ assume i hi, (hf i hi).has_sum).summable
 
+lemma has_sum.add_disjoint {s t : set β} (hs : disjoint s t)
```

Comment:
Should this include `subtype` in the name, to make it clear it has a coercion?

Reply:
Ah, I see plenty of lemmas below use `coe` in the same way and don't, so nevermind.